tests/logging/test-logging.go

```diff
@@ -0,0 +1,213 @@
+				return
+			}
+			log.Println("Waiting for OKLog pods to be READY!!")
+		}
+	}
+}
+
+func testOKLogUI() {
+	resp, err := http.Get("http://oklog:7650/ui/")
+
+	if err != nil {
+		log.Fatalf("Test Check OKLogUI Failed: error is: %v and response is: %v", err, resp)
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != 200 {
+		body, _ := ioutil.ReadAll(resp.Body)
+		log.Fatalf("Test Check OKLogUI Failed: Unable to fetch UI. Response code is: %v and response test is: %v", resp.StatusCode, string(body))
+	}
+	log.Printf("Test Check OKLogUI Passed. Response code is: %v", resp.StatusCode)
+}
+
+func testLogSpout() {
+	timeout := time.After(2 * time.Minute)
+	tick := time.Tick(1 * time.Second)
+	var testDataRegex = regexp.MustCompile(`(?m)oklog.kyma-system:7651*`)
+
+	for {
+		select {
+		case <-timeout:
+			cmd := exec.Command("kubectl", "-n", namespace, "logs", "-l", "app=logspout")
+			stdoutStderr, _ := cmd.CombinedOutput()
+			log.Fatal("Timed out getting the correct log for pod logspout", ":\n", string(stdoutStderr))
+		case <-tick:
+			cmd := exec.Command("kubectl", "-n", namespace, "logs", "-l", "app=logspout")
+			stdoutStderr, err := cmd.CombinedOutput()
+			if err != nil {
+				log.Fatal("Unable to obtain function log for pod logspout", ":\n", string(stdoutStderr))
+			}
+
+			submatches := testDataRegex.FindStringSubmatch(string(stdoutStderr))
+			if submatches != nil {
+				log.Printf("Test Check LogSpout has Passed. OKLog Service is set in logspout: %v", submatches)
+				return
+			}
+		}
+	}
+}
+
+func deployDummyPod() {
+	cmd := exec.Command("kubectl", "create", "-f", yamlFile)
+	stdoutStderr, err := cmd.CombinedOutput()
+	if err != nil {
+		log.Fatal("Unable to deploy:\n", string(stdoutStderr))
+	}
+}
+
+func waitForDummyPodToRun() {
+	timeout := time.After(2 * time.Minute)
+	tick := time.Tick(1 * time.Second)
+
+	for {
+		select {
+		case <-timeout:
+			log.Println("Test LogStreaming: result: Timed out!!")
+			cmd := exec.Command("kubectl", "get", "pods", "-l", "app=counterTestPod", "-n", namespace, "--no-headers")
+			stdoutStderr, _ := cmd.CombinedOutput()
+			log.Fatal("Test LogStreaming: result: Timed out!!. Current state is", ":\n", string(stdoutStderr))
+		case <-tick:
+			cmd := exec.Command("kubectl", "-n", namespace, "get", "pod", "counter-test-pod", "-ojsonpath={.status.phase}")
+			stdoutStderr, err := cmd.CombinedOutput()
+
+			if err == nil && strings.Contains(string(stdoutStderr), "Running") {
+				log.Printf("Returning")
+				return
+			}
+		}
+	}
+}
+
+func testLogs() {
+	log.Println("Starting")
+	c := &http.Client{
+		Timeout: 45 * time.Second,
+	}
+
+	res, err := c.Get("http://oklog:7650/store/stream?q=oklogTest-")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer res.Body.Close()
+	var testDataRegex = regexp.MustCompile(`(?m)oklogTest-*`)
+
+	log.Println("Here")
+
+	reader := bufio.NewReader(res.Body)
+	for {
+		line, err := reader.ReadBytes('\n')
+		if err != nil {
+			return
+		}
+		println(string(line))
```

Comment:
How does `println` work? I see no difference in output but there is no compilation error as well.

Reply:
removed